build/app.js

```diff
@@ -116,6 +117,11 @@ exports.execute = function(options) {
   var stageDir = utils.getFile(options.STAGE_DIR);
   utils.ensureFolderExists(stageDir);
 
+  var l10nTSPath = utils.joinPath(stageDir.path, 'timestamp-l10n.json');
+  if (utils.fileExists(l10nTSPath)) {
+    utils.deleteFile(l10nTSPath);
+  }
+
```

Comment:
So, my logic her is:
 - I want to put a timestamp in HTML files based on latest revision from hg/bit repo
 - it is the same timestamp in every app so I only need to read it for one app and then I want to cache it and use it in the rest
 - but between two builds, user can update their locale repositories so I want to reset the timestamps cache file at the beginning of the build.

The scenario I'm preventing with this cache invalidation is this:

 - user clones a locale
 - user builds Gaia with a timestamp from that locale
 - we cache the timestamp
 - user updates locale repo
 - user builds Gaia again

Without this deletion, Gaia build system will pick up the timestamp from cache which is from the previous build and is an old timestamp.

My goal is to restart timestamps each time we launch build script.

Does it make sense? I may be misunderstanding how build system works.